<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectWizard</class>
 <widget class="QWizard" name="NewProjectWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>382</width>
      <height>239</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create Project at selected Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="ErrorIcon">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap">
           <pixmap resource="style.qrc">:/qss_icons/rc/ErrorIcon.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Information">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ProjectName">
       <item>
        <widget class="QLabel" name="ProjectNameLabel">
         <property name="text">
          <string>Project name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ProjectNameEntry"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="DefaultPathButton">
          <property name="text">
           <string>&amp;Use default location</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ProjectPath">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="ProjectPathLabel">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ProjectPathEntry">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ProjectPathBrowsButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="Spacing" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage2"/>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
